# HG changeset patch
# User Benjamin Lindner <lindnerb@users.sourceforge.net>
# Date 1250187632 -7200
#      Thu Aug 13 20:20:32 2009 +0200
# Node ID d280bfa04996fa3bf2e1d02f22dea56fdb6d6d62
# Parent  163e205600a4283ba5d41b165fe93dced01c6823
skip double declaration of tztime

diff --git a/liboctave/strftime.c b/liboctave/strftime.c
--- a/liboctave/strftime.c
+++ b/liboctave/strftime.c
@@ -44,17 +44,17 @@
 #else
 # ifdef HAVE_SYS_TIME_H
 #  include <sys/time.h>
 # else
 #  include <time.h>
 # endif
 #endif
 
-#if HAVE_TZNAME
+#if HAVE_TZNAME && !HAVE_DECL_TZNAME
 extern OCTAVE_IMPORT char *tzname[];
 #endif
 
 /* Do multibyte processing if multibytes are supported, unless
    multibyte sequences are safe in formats.  Multibyte sequences are
    safe if they cannot contain byte sequences that look like format
    conversion specifications.  The GNU C Library uses UTF8 multibyte
    encoding, which is safe for formats, but strftime.c can be used
